src/Misc/layoutbin/installdependencies.sh

```diff
@@ -257,6 +256,28 @@ then
                 print_errormessage
                 exit 1
             fi
+        elif [ -e /etc/mariner-release ]
+        then
+            echo "The current OS is Mariner based"
```

Comment:
On [line 10](https://github.com/microsoft/azure-pipelines-agent/pull/3698/files#R10), there's a small description of how different OSs are detected. Could you please also add a note about Mariner OS there?